tests/backup/backup_helper.go

```diff
@@ -1473,12 +1473,17 @@ func GetAllRestoresNonAdminCtx(ctx context.Context) ([]string, error) {
 }
 
 // DeletePodWithLabelInNamespace kills pod with the given label in the given namespace
-func DeletePodWithLabelInNamespace(namespace string, label map[string]string) error {
+func DeletePodWithLabelInNamespace(namespace string, label map[string]string, ignoreLabel bool) error {
```

Comment:
```go
if ignoreLabel {
    label = make(map[string]string)
}
```
?

Reply:
Can't do this as we still need label at the end to do filter

Reply:
We need the label later to not delete the other pods